modules/400-descheduler/crds/deschedulers.yaml

```diff
@@ -0,0 +1,584 @@
+                                    matchExpressions:
+                                      items:
+                                        properties:
+                                          key:
+                                            type: string
+                                          operator:
+                                            type: string
+                                          values:
+                                            items:
+                                              type: string
+                                            type: array
+                                        required:
+                                          - key
+                                          - operator
+                                        type: object
+                                      type: array
+                                    matchLabels:
+                                      additionalProperties:
+                                        type: string
+                                      type: object
+                                  type: object
+                                  x-kubernetes-map-type: atomic
+                                namespaces:
+                                  properties:
+                                    exclude:
+                                      items:
+                                        type: string
+                                      type: array
+                                    include:
+                                      items:
+                                        type: string
+                                      type: array
+                                  type: object
+                                nodeFit:
+                                  type: boolean
+                                removeFailedPods:
+                                  properties:
+                                    excludeOwnerKinds:
+                                      items:
+                                        type: string
+                                      type: array
+                                    includingInitContainers:
+                                      type: boolean
+                                    minPodLifetimeSeconds:
+                                      type: integer
+                                    reasons:
+                                      items:
+                                        type: string
+                                      type: array
+                                  type: object
+                                thresholdPriority:
+                                  format: int32
+                                  type: integer
+                                thresholdPriorityClassName:
+                                  type: string
+                              type: object
+                          type: object
+                        removePodsHavingTooManyRestarts:
```

Comment:
Do we need all these strategies, or can we go with a limited number for a start? I think it can make the review of this PR much easier.

Reply:
The only strategies supported are the ones that were supported before.